scene/2d/tile_map.cpp

```diff
@@ -1144,6 +1144,21 @@ Array TileMap::get_used_cells() const {
 	return a;
 }
 
+Array TileMap::get_used_cells_by_id(int p_id) const {
+
+	Array a;
+	a.clear();
```

Comment:
I guess the `a.clear()` is not necessary here, `Array a;` already declares it empty.